Assets/Scripts/Game/Questing/Actions/JournalNote.cs

```diff
@@ -49,7 +49,7 @@ public override void Update(Task caller)
         {
             Message message = ParentQuest.GetMessage(id);
 
-            GameManager.Instance.PlayerEntity.Notebook.AddNote(new List<TextFile.Token>(message.GetTextTokens()));
+            GameManager.Instance.PlayerEntity.Notebook.AddNote(new List<TextFile.Token>(message.GetTextTokens(-1, false)));
```

Comment:
Maybe you could use named parameters for readabililty. `message.GetTextTokens(variant:-1, expandMacros:false)`

Reply:
Good idea, I'll update the PR.